libs/coin-tester/README.md

```diff
@@ -65,6 +65,8 @@ SEED=chronic find success crew board merit elder life achieve scout gravity soul
 GH_TOKEN=
 SPECULOS_API_PORT=4040
 SPECULOS_IMAGE=speculos
+# Only for CoinTester Bitcoin
+BITCOIN_REGTEST=on
```

Comment:
I don't like putting bitcoin specific variable here, maybe we can just rename to COIN_TESTER_ENV or something